microprofile/server/src/test/java/io/helidon/microprofile/server/StreamingOutputLeakTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.microprofile.server;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.QueryParam;
+import jakarta.ws.rs.client.WebTarget;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.core.StreamingOutput;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Random;
+
+import io.helidon.microprofile.tests.junit5.AddBean;
+import io.helidon.microprofile.tests.junit5.AddConfig;
+import io.helidon.microprofile.tests.junit5.AddExtension;
+import io.helidon.microprofile.tests.junit5.DisableDiscovery;
+import io.helidon.microprofile.tests.junit5.HelidonTest;
+
+import org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.CoreMatchers.is;
+
+@HelidonTest
+@DisableDiscovery
+@AddBean(StreamingOutputLeakTest.DownloadResource.class)
+@AddExtension(ServerCdiExtension.class)
+@AddExtension(JaxRsCdiExtension.class)
+@AddExtension(CdiComponentProvider.class)
+@AddConfig(key = "server.backpressure-buffer-size", value = "20971520")//20Mb
+public class StreamingOutputLeakTest {
+
+    private static final int SIZE10MB = 10 * 1024 * 1024;
+    private static final int SIZE = SIZE10MB;
+    private static final long NUMBER_OF_BUFS = 200;
+    private static final byte[] DATA_10MB = new byte[SIZE];
+
+    static {
+        Random r = new Random();
+        r.nextBytes(DATA_10MB);
+    }
+
+    /**
+     * Reproducer for issue #4643
+     */
+    @Test
+    @Disabled("Takes too long (around 2 minutes), root-cause covered by WaterMarkedBackpressureTest")
+    void streamingOutput(WebTarget target) throws IOException {
```

Comment:
The test is disabled and there is no other test in this class, so this class is practically useless.
This should not be included into Helidon.
Either we should come up with an integration tests profile for "longer" tests, or change the test to be usable.